src/core/playground/bcs.ts

```diff
@@ -0,0 +1,23 @@
+import { Serializer } from "../../bcs";
+import { Hex } from "../hex";
+import type { Serializable } from "./interfaces";
+
+/**
+ * Serializes a `Serializable` value to its BCS representation.
+ * This function is the Typescript SDK equivalent of `bcs::to_bytes` in Move.
+ * @returns the BCS representation of the Serializable instance as a byte buffer
+ */
+export function bcsToBytes(serializable: Serializable): Uint8Array {
```

Comment:
we already have these functions right?

Reply:
Yeah, just made them into pure functions instead of abstract class methods.
In general I personally prefer using interfaces and pure functions (it works better with duck-typed languages like python and javascript).

But talking offline with @xbtmatt I think there's a general preference on using a `Serializable` abstract class, with methods.

Reply:
yes, I like the doing `<>.bcsToBytes()` (where `<>` is anything extends Serializable